<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>423</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Path Info Table</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1250</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1250</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>501</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Schedule Trains</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Select Train</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="trainBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Select Destination</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="destinationBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="scheduleButton">
           <property name="text">
            <string>Schedule</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1076</x>
      <y>10</y>
      <width>201</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pavelow Connection Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="connection_statusLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_State"/>
    <addaction name="actionLoad_State"/>
   </widget>
   <widget class="QMenu" name="menuTrain">
    <property name="title">
     <string>Train</string>
    </property>
    <addaction name="actionAdd_Train"/>
    <addaction name="actionEdit_Train"/>
    <addaction name="actionDelete_Train"/>
   </widget>
   <widget class="QMenu" name="menuTables">
    <property name="title">
     <string>Tables</string>
    </property>
    <widget class="QMenu" name="menuLocal_Tables">
     <property name="title">
      <string>Local Tables</string>
     </property>
     <addaction name="actionView_Trains"/>
     <addaction name="actionView_DS_Connectivity"/>
     <addaction name="actionView_Pathinfo"/>
     <addaction name="actionView_Trackinfo"/>
     <addaction name="actionView_Tracklisting"/>
    </widget>
    <widget class="QMenu" name="menuRemote_Tables">
     <property name="title">
      <string>Remote Tables</string>
     </property>
     <addaction name="actionView_Schedule_train_info"/>
     <addaction name="actionView_Scheduled_routes"/>
     <addaction name="actionView_Scheduled_train_info"/>
    </widget>
    <addaction name="menuLocal_Tables"/>
    <addaction name="menuRemote_Tables"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTrain"/>
   <addaction name="menuTables"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_State">
   <property name="text">
    <string>Save State</string>
   </property>
  </action>
  <action name="actionLoad_State">
   <property name="text">
    <string>Load State</string>
   </property>
  </action>
  <action name="actionAdd_Train">
   <property name="text">
    <string>Add Train</string>
   </property>
  </action>
  <action name="actionEdit_Train">
   <property name="text">
    <string>Edit Train</string>
   </property>
  </action>
  <action name="actionDelete_Train">
   <property name="text">
    <string>Delete Train</string>
   </property>
  </action>
  <action name="actionView_Trains">
   <property name="text">
    <string>View Trains</string>
   </property>
  </action>
  <action name="actionView_DS_Connectivity">
   <property name="text">
    <string>View DS_Connectivity</string>
   </property>
  </action>
  <action name="actionView_Tracklisting">
   <property name="text">
    <string>View Tracklisting</string>
   </property>
  </action>
  <action name="actionView_Trackinfo">
   <property name="text">
    <string>View Trackinfo</string>
   </property>
  </action>
  <action name="actionView_Pathinfo">
   <property name="text">
    <string>View Pathinfo</string>
   </property>
  </action>
  <action name="actionView_Schedule_train_info">
   <property name="text">
    <string>View Schedule_train_info</string>
   </property>
  </action>
  <action name="actionView_Scheduled_routes">
   <property name="text">
    <string>View Scheduled_routes</string>
   </property>
  </action>
  <action name="actionView_Scheduled_train_info">
   <property name="text">
    <string>View Scheduled_train_info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
